scripts/test

```diff
@@ -23,6 +23,9 @@ fi
 
 set_project_vars
 
+#a dummy cadvisor for testing multiple container stats
+docker run --detach=true google/cadvisor:latest
```

Comment:
This won't scale well as we add more tests to host-api (something I really want to do now that we have a start).
I think the right thing to do is to start and destroy the test containers in the acttual test. The paradigm established in python-agent is a good one to follow: each test is responsible for creating the container(s) it needs to test the feature.